### DISCUSSION TITLE
Make long tooltips span multiple lines

### DISCUSSION THREAD

**Discussion by CrabbyDisk** [picocss/pico#639] [Ideas]
When a tooltip gets long, it can be hard to read and if it's close to the edge of the screen, it will get cut off. Maybe the width of tooltips could be limited, or there could be a way to directly add line breaks manually.

**Comment by Yohn** [comment]
`&#xa;` I think could make the new lines

**Comment by Yohn** [comment]
Ok, so there was a little more that needed to be done for this to work.

```css
display: inline-block;
white-space: pre;
```
needed to be added to the `::before` pseudo-element. You can see the updated [_tooltip.scss here](/Yohn/PicoCSS/blob/main/scss/components/_tooltip.scss) along with a [demo of the multi-line tooltip](https://yohn.github.io/PicoCSS/)

**Comment by Kugeleis** [comment]
Is there a way to do this without forking the entire project?

**Reply by Yohn** [reply]
`npm install @yohns/picocss` 

^ is one way to install from my from, or I have the cdn links updated in the [Yohn/PicoCSS Read Me](https://github.com/Yohn/PicoCSS)
